internal/crossdock/client/dispatcher/dispatcher.go

```diff
@@ -39,13 +40,25 @@ import (
 
 // Create creates an RPC from the given parameters or fails the whole behavior.
 func Create(t crossdock.T) *yarpc.Dispatcher {
+	return CreateTransport(t, "")
+}
+
+// CreateTransport creates an RPC from the given parameters or fails the whole behavior.
+//
+// If trans is non-empty, this will be used instead of the behavior transport.
+func CreateTransport(t crossdock.T, trans string) *yarpc.Dispatcher {
```

Comment:
CreateDispatcherForTransport?

Transport is already a top-level object, so it's a bit confusing that a "CreateTransport" function returns a dispatcher.